From 62e178ddae24c7633d2a03b129dc52261e0a15bf Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 7 Feb 2018 00:48:03 -0800
Subject: [PATCH] Define MESA_EGL_NO_X11_HEADERS before including eglplatform.h
 when X11 is disabled

This helps in compiling when GL implementations e.g. mesa use
this define to exclude X11 headers

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Submitted [https://github.com/anholt/libepoxy/pull/153]

 src/dispatch_common.h | 3 +++
 1 file changed, 3 insertions(+)

Index: libepoxy-1.4.3/src/dispatch_common.h
===================================================================
--- libepoxy-1.4.3.orig/src/dispatch_common.h
+++ libepoxy-1.4.3/src/dispatch_common.h
@@ -46,6 +46,9 @@
 #include "epoxy/glx.h"
 #endif
 #if PLATFORM_HAS_EGL
+#if !PLATFORM_HAS_GLX
+#define MESA_EGL_NO_X11_HEADERS 1
+#endif
 #include "epoxy/egl.h"
 #endif
 #if PLATFORM_HAS_WGL
